webrtc-stats.html

```diff
@@ -1617,6 +1618,18 @@ <h2>
                   packetize the resulting data.
                 </p>
               </dd>
+              <dt>
+                <dfn><code>totalPacketSendDelay</code></dfn> of type <span class=
+                "idlMemberType">double</span>
+              </dt>
+              <dd>
+                <p>
+                  The total number of seconds that packets have spent buffered locally before being
+                  transmitted onto the network. Measurement starts when a packet has been produced
+                  and ends when the packet is sent. <code>totalPacketSendDelay</code> is incremented
```

Comment:
Needs to define what "is sent" means - probably it is when the send() call is executed on the actual socket - which is after the webrtc congestion control, but before the tcp congestion control when the socket is a TCP socket. "last possible time to report before the clock is sent" would do.